Scheduler/observing_blocks_maintel/BLOCK-T74/BLOCK-T74_1.json

```diff
@@ -0,0 +1,412 @@
+{
+    "name": "BLOCK-T74_1",
+    "id": "e7afd17e-a473-4890-a2e0-510582b318ad",
```

Comment:
We does this come from?  (I remember we talked about this but can't remember...)

Reply:
This was automatically generated by the ts_observing package when I created the block.

Reply:
If a user doesn't provide it, it is generated by the scheduler CSC on execution.